tests/openvino/utils_tests.py

```diff
@@ -72,6 +72,8 @@
     "gpt_neox": "hf-internal-testing/tiny-random-GPTNeoXForCausalLM",
     "gpt_neox_japanese": "hf-internal-testing/tiny-random-GPTNeoXJapaneseForCausalLM",
     "gptj": "hf-internal-testing/tiny-random-GPTJModel",
+    "granite": "katuni4ka/tiny-random-granite",
```

Comment:
I'd include a test for these models in https://github.com/huggingface/optimum-intel/blob/bb1c68ae3da29ae223ccd3a6ac2635c80a0539d1/tests/openvino/test_modeling.py#L860

Reply:
Oops.., it looks like I accidentally forgot to push this file update :)

Reply:
added